dj/api/nodes.py

```diff
@@ -207,6 +225,25 @@ def read_node(
     return node  # type: ignore
 
 
+@router.get("/nodes/{name}/revisions/", response_model=List[NodeRevisionMetadata])
```

Comment:
I know we're calling these revisions in the source code, but would this be more intuitive if it was "versions" here? Just the endpoint I mean, as in `"/nodes/{name}/versions/"`

Reply:
I think this is a minor issue, but imo `revision` seems to be more technical term and I think it helps to see it as an internal feature (for change tracking) rather than external feature for supporting multiple versions of something.

Reply:
Agreed, I definitely don't think this should block this PR. I'll go ahead and merge this in.